src/bootstrap.coffee

```diff
@@ -111,7 +111,10 @@ bootstrapper.startBootstrapping = ->
 		readConfigAndEnsureUUID()
 		.tap ->
 			loadPreloadedApps()
-		.tap ->
-			bootstrapOrRetry()
+		.tap (uuid) ->
+			if bootstrapper.offlineMode
+				return knex('config').insert({ key: 'uuid', value: uuid })
```

Comment:
So to make sure I understand this change is only for that check, ie to avoid reading config.json for the uuid when the supervisor gets restarted whilst in offline mode?